worker/edge/go.mod

```
@@ -0,0 +1,3 @@
+module github.com/joschahenningsen/tum-live-worker-v2/edge
+
+go 1.17
```

```suggestion
go 1.18
```

Not sure if this makes a difference, but if were using workspaces, it could make sense to update everything to 1.18 just to be sure.